### ISSUE TITLE
copy() error: 401 Unauthorized 

### ISSUE THREAD

**Issue by nikitul** [russplaysguitar/CouchDB-for-Coldfusion#3]
Hello,

I played a little with your wrapper and CouchDB 1.6.1,  and I get "copy() error: 401 Unauthorized " error on document_test.cfm when this tries to copy a document.
this is because user/password is not sended in document.cfc Net_Socket().
I resolved this problem by changing this piece of code from documentcfc copy()

``` coldfusion
        <!--- custom HTTP Request --->
        <!--- note: this is sensitive to formatting. do not add tabs. --->
        <cfset http_request[1] = "COPY /#variables.db_name#/#arguments.id# HTTP/1.1">
        <cfset http_request[2] = "Destination: #arguments.new_id##revision_string#">
```

with this

``` coldfusion
        <!--- encode base 64 for user / password --->
        <CFSET var credentials = binaryEncode(
            toBinary(
                toBase64( "#variables.couch_username#:#variables.couch_password#" )
            )
        , "base64" )>

        <!--- custom HTTP Request --->
        <!--- note: this is sensitive to formatting. do not add tabs. --->
        <cfset http_request[1] = "COPY /#variables.db_name#/#arguments.id# HTTP/1.1">
        <cfset http_request[2] = "Authorization: Basic " & credentials>
        <cfset http_request[3] = "Destination: #arguments.new_id##revision_string#">
```

**Comment by russplaysguitar** [comment]
Cool, thanks for the fix. Are you using the latest version of CouchDB? Also: are you able to contribute your changes by a pull-request?

**Comment by nikitul** [comment]
I will try :)
Is my first one :)

**Comment by nikitul** [comment]
I think I succeeded to pull a request.
It is a little modified version but not to much.

**Comment by russplaysguitar** [comment]
Fixed by: #5
